### PR TITLE
add doc for local deployment

### PR DESCRIPTION
**Pull request by messixukejia**

### CODE REVIEW

File: docs/zh/quickstart/How-to-local-deploy.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
+阿里已经正式[开源](https://mp.weixin.qq.com/s/hcCYBZ0qvs8q4Wp1zFt-lg)了可观测数据采集器iLogtail。作为阿里内部可观测数据采集的基础设施，iLogtail承载了阿里巴巴集团、蚂蚁的日志、监控、Trace、事件等多种可观测数据的采集工作。
```
Comment (EvanLjp) [inline docs/zh/quickstart/How-to-local-deploy.md:1]: How about making a release doc folder to contain the docs？ 
such as， `docs/zh/usecases` ?

File: docs/zh/quickstart/How-to-local-deploy.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+​
+
```
Comment (EvanLjp) [inline docs/zh/quickstart/How-to-local-deploy.md:3]: Please make a bookmark

**Review by EvanLjp — CHANGES_REQUESTED** [review]

File: docs/zh/quickstart/How-to-local-deploy.md (hunk 1 above)
Comment (messixukejia) [reply docs/zh/quickstart/How-to-local-deploy.md]: done

File: docs/zh/quickstart/How-to-local-deploy.md (hunk 2 above)
Comment (messixukejia) [reply docs/zh/quickstart/How-to-local-deploy.md]: done

**Pull request merged**